### DISCUSSION TITLE
Page not found after update

### DISCUSSION THREAD

**Discussion by geckofish** [teslamate-org/teslamate#4126] [Q&A]
I did a pull of 1.30.1.  After that, I get a "404 page not found".  I can still access grafana, but the data is not updating.  I've tried restarting and rebooting the server.  The log looks like this.  Any ideas how to fix it?

teslamate_1  | Aborted (core dumped)
teslamate_1  | waiting for postgres at database:5432
teslamate_1  | Failed to create thread: Operation not permitted (1)
teslamate_1  | Aborted (core dumped)

**Comment by geckofish** [comment]
I downgraded to 1.28.3, and things seem to be working again.  I'm on debian stretch, so I think I need to update before upgrading teslamate.

**Comment by brianmay** [comment]
See #3754
